drivers/sensor/icm42688/icm42688_decoder.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_ICM42688_DECODER_H_
+#define ZEPHYR_DRIVERS_SENSOR_ICM42688_DECODER_H_
+
+#include <stdint.h>
+#include <zephyr/drivers/sensor.h>
+
+struct icm42688_decoder_header {
+	uint8_t is_fifo: 1;
+	uint8_t gyro_fs: 3;
+	uint8_t accel_fs: 2;
+	uint8_t reserved: 2;
+	uint64_t timestamp;
```

Comment:
Will this be okay with this not being 64bit aligned?  I can see this as the first item of the struct.

Reply:
Makes sense, moved.